public/docs/ts/latest/cookbook/a1-a2-quick-reference.jade

```diff
@@ -510,14 +504,13 @@ table(width="100%")
       code-example.
         &lt;tr ng-repeat="movie in movieList | filter: {title:listFilter}">
       :marked
-        Selects a subset of items from the defined collection based on the filter criteria.
+        Selects a subset of items from the defined collection, based on the filter criteria.
     td
       :marked
         ### none
-        There is no comparable pipe in Angular&nbsp;2 for performance reasons.
-        Filtering should be coded in the component.
-        Consider building a custom pipe if the same filtering code
-        will be reused in several templates.
+        For performance reasons, no comparable pipe exists in Angular&nbsp;2.
+        Code filters in the component. If the same filtering code
```

Comment:
"Code filters in the component" sounded like a noun phrase to me. Clarify, so it's clear who does what.